### DISCUSSION TITLE
Unable to open search result item into a new tab from Card view

### DISCUSSION THREAD

**Discussion by kuuwakio** [microsoft-search/pnp-modern-search#1452] [Q&A]
Is there a way to open a search result item into a new tab? using target="_blank" AND data-interception=off has no effect as the field doesn't understand HTML.

I'm using the latest version of Modern Search v4 and the Cards view, where I'd like to get the files within search results to open to new tab which it does not do. Documents open to the same tab and this is not the wished outcome.

**Comment by JoeAyre** [comment]
Seems to be similar behaviour to the 'Details list' layout. I've added a separate question for this (https://github.com/microsoft-search/pnp-modern-search/discussions/2654). Wonder if anyone can suggest a way to force items opening in a new tab. Using the 'List' layout allows items to open in a new tab so not really sure why this behaviour isn't replicated in 'Details list' (or 'Cards') layouts. @wobba - wonder if you might be able to shed some light on this? Thanks in advance. Joe

**Reply by wobba** [reply]
#2654 is different as you can control the markup. When I click the item title in card view it does open in a new tab. We can add the data-interception tag as well. How do you repro?

For the location part you can change the location card field to:

```
<a data-interception="off" target="_blank" style="color:{{@root.theme.palette.themePrimary}};font-weight:600;font-family:'{{@root.theme.fonts.small.fontFamily}}'" href="{{SPSiteUrl}}">{{SiteTitle}}</a>
```
